### ISSUE TITLE
Error "Invalid left-hand side in arrow function parameters" on QChatMessage

### ISSUE THREAD

**Issue by youssef11420** [quasarframework/quasar#9902]
Hello,

**Describe the bug**
When I upgraded quasar app to 2.2.10, I face an error when building or launching app with vue cli:

```
 ERROR  Failed to compile with 1 error                                                                                                                                                                          15:00:32
 error  in ./node_modules/quasar/src/components/chat/QChatMessage.js

Syntax Error: Unexpected token, expected , (89:54)


 @ ./node_modules/quasar/src/components/chat/index.js 27:0-45 28:0-24
 @ ./node_modules/quasar/src/components.js
 @ ./node_modules/quasar/src/index.esm.js
 @ ./src/main.js
 @ multi ./node_modules/webpack-dev-server/client?http://10.179.39.234:8080&sockPath=/sockjs-node (webpack)/hot/dev-server.js ./src/main.js
```

**Codepen/jsFiddle/Codesandbox (required)**
It's in my main.js, I can't use a Codepen/jsFiddle/Codesandbox.

**To Reproduce**

**Expected behavior**

**Screenshots**

**Platform (please complete the following information):**
Quasar Version: 1.15.21
@quasar/app Version: 2.2.10

Quasar mode:
  - [x] SPA
  - [ ] SSR
  - [ ] PWA
  - [ ] Electron
  - [ ] Cordova
  - [ ] Capacitor
  - [ ] BEX

Tested on:
  - [x] SPA
  - [ ] SSR
  - [ ] PWA
  - [ ] Electron
  - [ ] Cordova
  - [ ] Capacitor
  - [ ] BEX

OS:
Node: v14.17.1
NPM: -
Yarn: v1.22.10
Browsers: -
iOS: -
Android: -
Electron: -

**Additional context**
I can fix the bug when modifying the QChatMessage source (lines 88-90):

Buggy code:
```
const content = withSlots === true
        ? (contentList.length > 1 ? text => text : text => h('div', [ text ]))
        : text => h('div', { domProps: { [domPropText]: text } })
```

Fixed code:
```
const content = withSlots === true
        ? (contentList.length > 1 ? (text => text) : (text => h('div', [ text ])))
        : (text => h('div', { domProps: { [domPropText]: text } }))
```

Can you please fix that?

Thanks in advance.

Youssef

**Comment by rstoenescu** [comment]
"@quasar/app" has nothing to do with Vue CLI.
Regardless, merged your PR and will be available in next "quasar" v1.x.

**Comment by youssef11420** [comment]
Hello @rstoenescu,

Can you please tell me when will you make the next v1 version available in npm registry?

Thanks in advance.

Youssef

**Comment by rstoenescu** [comment]
@youssef11420 released it just now (v1.15.22)
